NickvisionMoney.Shared/Resources/Strings.nl.resx

```diff
@@ -525,10 +525,10 @@ Let op: niet-opgeslagen aanpassingen wordt gewist.</value>
   <data name="AccountFileFilter.GTK" xml:space="preserve">
 		<value>Money-rekening (*.nmoney)</value>
   </data>
+  <data name="MetaDescription.GTK" xml:space="preserve">
+		<value></value>
+  </data>
   <data name="Overview.Today" xml:space="preserve">
     <value>Dagoverzicht</value>
   </data>
-  <data name="DeleteTransaction.SourceRepeat" xml:space="preserve">
-    <value>Transactie verwijderen</value>
```

Comment:
Why was this deleted?

Reply:
I didn't touch it, guess git decided to remove it without notifying me when I was resolving conflicts